src/NuoJsConnection.cpp

```diff
@@ -440,6 +439,7 @@ NuoDB::PreparedStatement* Connection::createStatement(std::string sql, Local<Arr
             // fundamental data types in ES. Within these types we need to
             // check if it will result in a safe conversion (e.g. Number).
             switch (sqlType) {
+                case NuoDB::NUOSQL_UNDEFINED:
```

Comment:
Why this change?

Reply:
I agree that undefined maps more naturally to null than to string :)

Does that make more sense than throwing an exception?